examples/evaluation_and_profiling/simple_calculator_eval/harbor_adapters/template/tests/evaluate.py

```
@@ -0,0 +1,105 @@
+
+    Returns:
+        Process exit code where 0 means the answer passed verification.
+
+    Raises:
+        OSError: If verifier artifact files cannot be written.
+    """
+    try:
+        ground_truth = _load_ground_truth()
+        expected_value = float(ground_truth["expected_value"])
+        tolerance = float(ground_truth.get("tolerance", 1e-4))
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)